deepspeed/inference/v2/ragged/sequence_descriptor.py

```diff
@@ -168,7 +168,9 @@ def cur_allocated_blocks(self, cache_group: int = 0) -> int:
         Arguments:
             cache_group (int): The cache group to query.
         """
-        return self._blocks_per_allocation_group[cache_group].sum()
+        if len(self._blocks_per_allocation_group) == 1:
+            return self._blocks_per_allocation_group[0].item()
```

Comment:
Is the torch tensor always going to be a single value in this branch? Is there a missing `sum()`?